desktop/core/src/desktop/js/apps/notebook2/components/aceEditor/autocomplete/AutocompleteResults.ts

```diff
@@ -653,7 +655,7 @@ class AutocompleteResults {
 
     let tableSuggestions: Suggestion[] = [];
     if (
-      this.dialect() === DIALECT.impala &&
+      this.dialect() === IMPALA_DIALECT &&
```

Comment:
For the record (I could not get it quickly), what is this specific case for Impala doing?

Reply:
So this an interesting one, consider "select * from a.b" in Impala "a" and "b" could either be a database and a table or a table and a struct. The parser can't figure this out as you'd need info about the schema so for the case "select * from a.|" the parser assumes "suggestTables" but it could be "suggestColumns", this code block checks if "a" is a known database to determine which one.